.env

```diff
@@ -0,0 +1,5 @@
+MAILER_DOMAIN=smtp.mailtrap.io
```

Comment:
You should have added this before to the .gitignore so that this file hadn't been uploaded

Reply:
Done!

Reply:
I deleted .env